app/ante/ante.go

```diff
@@ -1,6 +1,7 @@
 package ante
 
 import (
+	errorsmod "cosmossdk.io/errors"
```

Comment:
Shall we just use it as `"errors"`, only rename it when there's ambiguity?

Reply:
Done